### PR TITLE
#2652

### PR DESCRIPTION
**Pull request by vestrada**
https://github.com/craftercms/craftercms/issues/2652 - [studio] The box-file-upload control should save a /remote-assets/box style URL #2652

### CODE REVIEW

File: static-assets/components/cstudio-common/common-api.js [hunk 1]
```diff
@@ -5820,6 +5823,24 @@ var nodeOpen = false,
...
+                    success: function(response) {
+                        var contentResults = eval("(" + response.responseText + ")");
+                        callback.success(contentResults);
+                    },
+
+                    failure: function(response) {
+                        callback.failure(response);
+                    }
+                };
+
```
Comment (avasquez614) [inline static-assets/components/cstudio-common/common-api.js]: Remove innecessary extra space?

**Review by avasquez614 — CHANGES_REQUESTED** [review]

File: static-assets/components/cstudio-common/common-api.js [hunk 2]
```diff
@@ -5820,6 +5823,24 @@ var nodeOpen = false,
                 YConnect.asyncRequest("GET", this.createServiceUri(serviceUri), serviceCallback);
             },
 
+            getboxURL: function(site, profileId, fileId, callback) {
```
Comment (avasquez614) [inline static-assets/components/cstudio-common/common-api.js]: Should be `getBoxURL`

File: static-assets/components/cstudio-common/common-api.js (hunk 1 above)
Comment (vestrada) [reply static-assets/components/cstudio-common/common-api.js]: Done

File: static-assets/components/cstudio-common/common-api.js (hunk 2 above)
Comment (vestrada) [reply static-assets/components/cstudio-common/common-api.js]: Done

**Pull request merged**